tests/unit/core/server/testServer.js

```diff
@@ -180,6 +195,12 @@ describe(__filename, () => {
         .end();
 
       expect(response.statusCode).toEqual(500);
+      mockUserApi.verify();
+
+      // This ensures the error caught is the expected one, that is the one
```

Comment:
"caught is the expected one, that is the one" should be simplified to "caught is the expected one, from"

Reply:
![yoda](https://user-images.githubusercontent.com/1514/30284999-92fee7d8-9714-11e7-966f-e5dc2d06ce05.gif)